client/state/reader/follows/actions.js

```diff
@@ -196,3 +198,21 @@ export function unsubscribeToNewCommentEmail( blogId ) {
 		},
 	};
 }
+
+export function subscribeToNewPostNotifications( blogId ) {
```

Comment:
this would be a place we could add more info to.
```js
export function subsribeToNewPostNotification( site ) {
  return {
    type: READER_SUBSCRIBE_TO_NEW_POST_NOTIFICATIONS,
    payload: {
      site
   }
  }
}
```